com.vrcfury.vrcfury/Editor/VF/Service/ClipBuilderService.cs

```diff
@@ -171,10 +171,16 @@ public class ClipBuilderService {
                     foreach (var key in curve.FloatCurve.keys) {
                         (key.time == 0 ? startClip : endClip).SetConstant(binding, key.value);
                     }
+                    if (curve.FloatCurve.keys.Length == 1) {
+                        endClip.SetConstant(binding, curve.FloatCurve.keys[0].value);

```

Comment:
Is it possible for a binding to have 0 frames? Would it make more sense to change the foreach to
```
var first = false;
foreach (var key in curve.FloatCurve.keys) {
   if (first) {
     startClip.SetConstant(binding, key.value);
     first = false;
   }
   endClip.SetConstant(binding, key.value);
}
```

Reply:
I guess I'm also not sure what happens if the clip doesn't have a keyframe at time = 0. I'm also not sure if the frames are in time order 🤔

Reply:
That makes sense, I updated it with your snippet, that will cover more cases.

Reply:
seems like if there isn't a keyframe at time=0, time=0 inherits the first frame.